assets/src/data/eventespresso/schema/resolvers.js

```diff
@@ -231,8 +237,8 @@ export function* getRelationType( modelName, relationName ) {
  */
 export function* getRelationSchema( modelName, relationName ) {
 	modelName = singularModelName( modelName );
+	relationName = singularModelName( relationName );
 	const pluralRelationName = pluralModelName( relationName );
```

Comment:
this ok? just making sure nothing was missed

Reply:
Here it's okay too because a few lines below, it's being used to potentially retrieve the schema for the relation from the schema properties.  Similar to when relations are included in a response, the schema response may have relation records indexed by the singular form or the plural form of the relation.